bin/hyperlog.js

```diff
@@ -125,6 +149,22 @@ const main = async () => {
     if (cmd === 'read') {
       if (!key) throw new Error('ERR_KEY_REQUIRED')
 
+      const logConsole = argv.output ? argv.console : true
+      const logFile = argv.output ? fullPath(argv.output) : null
+      const inDir = argv['input-dir']
+        ? new RegExp('^' + escapeRegex(argv['input-dir']))
+        : null
+      const multiFileLog = !!inDir
+
+      if (logFile) {
+        const dirCreated = multiFileLog
+          ? await createDir(logFile)
+          : await createFileDir(logFile)
```

Comment:
If the path from `--output` matches an existing folder, the tool does not create a log file (if ` --input-dir` is specified)

Example:
- Create `out.log` folder
- Run `node ./bin/hyperlog.js read --key 0db350f414a93274e5613930b79c7226f21270cfc1435b96350422a0abf63dc2 --output out.log --console` in the first terminal
- Run `node ./examples/server.js` in the second terminal

Actual result: the logs file isn't created
Expected result: throw exception if folder already exists

Reply:
There is also other behavior when the flag `--input-dir` is specified, but I'm not sure if I use it correctly, since in my case a log file is created with the name as the log message

Reply:
re 1st one I think behavior should be like this:
- if logFile is directory and multiFileLog is true then it will demultiplex content based on start of line (tag) to each separate file (current behavior)
- if logFile is directory and multiFileLog is false then it should adjust logFile to be path.join(logFile, `hyperlog-${Date.now()}.log`
- if logFile is file and multiFileLog is true throw error because we don't know where to push files, e.g. parent dir of file?
- if logFile is file and multiFileLog is false then simply write to file (current behavior)

Reply:
Re 2nd one, the idea of input dir is to specify the part of starting line of multi file log that should be replaced, e.g.
```
/foo/bar/myFile-1.log >> ....
/foo/bar/myFile-2.log >> ....
```
in above case input-dir is `foo/bar/` and that one is created locally or replaced when `output` is specified

Reply:
https://github.com/vigan-abd/hypercore-logs/pull/1

Reply:
https://github.com/vigan-abd/hypercore-logs/pull/2